### DISCUSSION TITLE
How to load a script into my xkit

### DISCUSSION THREAD

**Discussion by angelinezhang21** [AprilSylph/XKit-Rewritten#1313] [Help Desk]
Sorry if this is a dumb question (I'm very new to Xkit), but I was intrigued by the Mute script [#677](https://github.com/AprilSylph/XKit-Rewritten/pull/677) and wasn't sure if I could add it to my browser Xkit to try out for tumblr? Thank you so much

**Comment by marcustyphoon** [comment]
Running development branches isn't too difficult if you're familiar with GitHub or can follow along with some instructions! I like to clone the repository with [GitHub Desktop](https://desktop.github.com/) (as it gives you an easy way to switch between branches). You can then load XKit Rewritten into your browser in development mode following the instructions linked in https://github.com/AprilSylph/XKit-Rewritten/blob/master/docs/Chapter%202%20-%20Getting%20started.md#loading-the-project-into-the-browser.

In Firefox, you'll have to do this each time you start your browser; in Chrome/Edge/Opera/etc it'll stay loaded. Just be aware that you'll have to pull any changes yourself (you won't automatically get any changes we make to any particular branch).
